Gemfile.lock

```diff
@@ -235,7 +248,7 @@ DEPENDENCIES
   tzinfo-data
 
 RUBY VERSION
-   ruby 2.6.4p104
+   ruby 2.6.5p114
```

Comment:
does this need to also be set in the Dockerfile?

Reply:
Yes it does. Although maybe it is done in a separate PR? I added a card on the triage https://trello.com/c/HcXuQxna/362-remove-ruby-version-definition-from-gemfilelock